### PR TITLE
[cli] Migrate `remove` command to new structure

### PR DESCRIPTION
**Pull request by trek**
Migrating `remove` to the command structure created in #10090. Unsure about tests. This is technically a refactor so existing tests should suffice.

### CODE REVIEW

**Review by cb1kenobi — APPROVED** [review]

File: packages/cli/src/commands/remove/command.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const removeCommand: Command = {
+  name: 'remove',
+  description: 'Remove a deployment by name or id',
```
Comment (cb1kenobi) [inline packages/cli/src/commands/remove/command.ts]: This should probably end with a period.

```suggestion
  description: 'Remove a deployment by name or id.',
```

**Review by cb1kenobi — CHANGES_REQUESTED** [review]

**Comment by trek** [comment]
Had to force push. Github got into some weird state where newer commits did not appear, but attempts to bring the branch up to date via the Github UI errored.

**Review by cb1kenobi — APPROVED** [review]

File: packages/cli/src/commands/remove/command.ts (hunk 1 above)
Comment (EndangeredMassa) [reply packages/cli/src/commands/remove/command.ts]: @cb1kenobi I don't think this was a blocking comment

Comment (cb1kenobi) [reply packages/cli/src/commands/remove/command.ts]: Correct, however consistency is king.

**Pull request merged**